op-guide/dashboard-tikv-info.md

```diff
@@ -248,7 +248,7 @@ RocksDB - kv | SST read duration | The time consumed when reading SST files |
 RocksDB - kv | Write stall duration | The time which is caused by write stall | It should be `0` in normal case.
 RocksDB - kv | Memtable size | The memtable size of each column family |
 RocksDB - kv | Memtable hit | The hit rate of memtable |
-RocksDB - kv | Block cache size | The block cache size of each column family |
+RocksDB - kv | Block cache size | The block cache size. Breakdown by column family if shared block cache is disabled. |
```

Comment:
The "Breakdown by" here is confusing for me. Could you explain more about it? A noun here looks weird.

Reply:
@lilin90 thanks for helping! For the "breakdown", let me give you an example. When shared cache is used, Grafana will show a single cache size, label with "all":
<img width="998" alt="Screen Shot 2019-05-08 at 4 41 29 PM" src="https://user-images.githubusercontent.com/2606959/57473837-a0f17780-7245-11e9-812e-7bc05975ca79.png">
when shared cache is disabled, Grafana will show individual cache size for each CFs:
<img width="1010" alt="Screen Shot 2019-05-09 at 10 46 50 AM" src="https://user-images.githubusercontent.com/2606959/57474779-c54e5380-7247-11e9-997a-7822b5328e5d.png">

Reply:
@yiwu-arbug Got it. Thanks for your explanation!